architecture/core_concepts/routes.adoc

```diff
@@ -322,9 +322,19 @@ In order for services to be exposed externally, an {product-title} route allows
 you to associate a service with an externally-reachable host name. This edge
 host name is then used to route traffic to the service.
 
-When two routes claim the same host, the oldest route wins. If additional routes
-with different path fields are defined in the same namespace, those paths will be
-added. If multiple routes with the same path are used, the oldest takes priority.
+When multiple routes from different namespaces claim the same host,
```

Comment:
This code seems not to implement that:
  https://github.com/knobunc/origin/blob/master/pkg/router/controller/unique_host.go#L106

And allowing someone else to inject a distinct path seems scary too.  If I claimed the path '/' and was serving the URL '/secret_thing' as one of my items, then someone else could put a path in with the same hostname and a more specific path underneath my general one and steal traffic.